src/ucp/wireup/wireup_cm.c

```diff
@@ -1266,6 +1272,12 @@ ucp_ep_cm_server_create_connected(ucp_worker_h worker, unsigned ep_init_flags,
         goto err_destroy_ep;
     }
 
+    cm_wireup_ep = ucp_ep_get_cm_wireup_ep(ep);
+    ucs_assert(cm_wireup_ep != NULL);
+    ucp_wireup_dst_rsc_indices_fill(ep, &ucp_ep_config(ep)->key, remote_addr,
```

Comment:
why need to do it here?
ucp_ep_create_to_worker_addr() already calls ucp_wireup_init_lanes() which calls ucp_wireup_check_config_intersect()

Reply:
added the comment